src/integrations/feed/configs.ts

```diff
@@ -226,19 +226,8 @@ export class FeedLofnConfigGenerator implements FeedConfigGenerator {
           this.feedPreferencesConfigGenerator.generate(ctx, opts),
         ]);
 
-        const configWithoutPagination = lofnConfig.config as Omit<
-          typeof lofnConfig.config,
-          'page_size' | 'total_pages'
-        > & {
-          page_size?: number;
-          total_pages?: number;
-        };
-
-        delete configWithoutPagination.page_size;
-        delete configWithoutPagination.total_pages;
```

Comment:
Just to make sure, these don't exist on lofn anymore so we don't have to delete them?

Reply:
Yeah @pnvasanth  is removing it here:
https://github.com/dailydotdev/lofn/pull/77